mod/groups/start.php

```diff
@@ -812,3 +815,60 @@ function groups_members_menu_setup($hook, $type, $menu, $params) {
 	
 	return $menu;
 }
+
+/**
+ * Prepare group profile buttons
+ * Returns an array of buttons as parameters suitable for ElggMenuItem::factory()
+ *
+ * @param string         $hook   "profile_buttons"
+ * @param string         $type   "group"
+ * @param ElggMenuItem[] $items  Buttons
+ * @param array          $params Hook params
+ * @return ElggMenuItem[]
+ */
+function groups_prepare_profile_buttons($hook, $type, $items, $params) {
+
+	$group = elgg_extract('entity', $params);
+	if (!$group instanceof ElggGroup) {
+		return;
+	}
+
+	$actions = [];
+	$buttons = [];
```

Comment:
Should we append items to `$items` in case someone wants to register items first?